features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -252,10 +257,17 @@ internal class DefaultImageWireframeHelper(
                     position = compoundDrawablePosition
                 )
 
+                wireframeIndex++
+                val resourceCacheKey = if (resourceIdCacheKey != null) {
+                    "$resourceIdCacheKey" + "_$wireframeIndex"
+                } else {
+                    null
+                }
```

Comment:
does it have any impact on the cache hit metric?

Reply:
I've done some measurements and the hit metric isn't impacted